### DISCUSSION TITLE
Restore Flipper support for tvOS

### DISCUSSION THREAD

**Discussion by philippbb** [react-native-tvos/react-native-tvos#337] [Feature Requests]
### Description

hi,

Sorry if this question has been asked before.

In the latest hello world template I don't find flipper anymore.
I was looking forward to use use_flipper_tvos! in the Podfile but it can't find this flipper command.

kind regards and thank you for an answer.

### Version

0.68.0

### Output of `npx react-native info`

OS: macOS 12.2
    CPU: (10) arm64 Apple M1 Pro
    Memory: 3.86 GB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.19.1 - /opt/homebrew/opt/node@14/bin/node
    Yarn: 1.22.18 - /opt/homebrew/bin/yarn
    npm: 6.14.16 - /opt/homebrew/opt/node@14/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.4, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
    Xcode: 13.3/13E113 - /usr/bin/xcodebuild
  Languages:
    Java: Not Found
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: Not Found
    react-native-macos: Not Found
    react-native-tvos:  0.68.0-0 
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

try to add use use_flipper_tvos! with 0.68

**Comment by douglowder** [comment]
Sadly, Flipper has a LOT of dependencies that are built for iOS only. I did get it to work, at considerable effort, in an earlier version of React Native, but haven't been able to get it to work since. I'll have a look at what the core team is doing in 0.68 and see if it's easier to make it work now.

**Reply by OrenMe** [reply]
Hi @douglowder, was there any development with this? Is it indeed easier now to get support?

**Comment by philippbb** [comment]
Thank you very much for your answer. I understand.

I was following allong with the 0.68 entry point from github but then somehow was not sure if tvos flipper should still work or I was doing something wrong.

**Comment by douglowder** [comment]
Status update: once Hermes is working with a 0.69 or 0.70 based release, I will attempt to restore support for Flipper on tvOS.

**Comment by douglowder** [comment]
Update on Flipper: There have been important recent developments in RN core, who are investigating the migration of RN debugging from Flipper to Chrome Dev Tools: https://twitter.com/preshonyee/status/1598372635309707277

For now, I am postponing work on Flipper. If it turns out that the core team does not move to Chrome Dev Tools, I will revisit this decision.
